src/main/scala/lenthall/util/ScalaConfig.scala

```diff
@@ -0,0 +1,49 @@
+package lenthall.util
+
+import com.typesafe.config.Config
+
+object ScalaConfig {
+
+  implicit class EnhancedScalaConfig(val config: Config) extends AnyVal {
+    def getStringOption(key: String): Option[String] = {
+      if (config.hasPath(key)) Option(config.getString(key)) else None
+    }
+
+    def getBooleanOption(key: String): Option[Boolean] = {
+      if (config.hasPath(key)) Option(config.getBoolean(key)) else None
+    }
+
+    def getIntOption(key: String): Option[Int] = {
+      if (config.hasPath(key)) Option(config.getInt(key)) else None
+    }
+
+    def getLongOption(key: String): Option[Long] = {
+      if (config.hasPath(key)) Option(config.getLong(key)) else None
+    }
+
+    def getDoubleOption(key: String): Option[Double] = {
+      if (config.hasPath(key)) Option(config.getDouble(key)) else None
+    }
+
+    def getStringOr(key: String, default: => String = ""): String = {
```

Comment:
I don't have strong opinions on this, it just seemed that the "option" methods were doing a chunk of the work the "or"s needed, but if you feel the costs outweigh the benefits that's ok. :smile: